datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.tsx

```diff
@@ -150,7 +150,7 @@ export const RECIPE_FIELDS = {
         ],
         filterFields: [TOPIC_ALLOW, TOPIC_DENY],
         advancedFields: [STATEFUL_INGESTION_ENABLED],
-        filterSectionTooltip:
+        sectionTooltip:
```

Comment:
I would still call this and all above `filterSectionTooltip` in case we want to add tooltips to any other sections as well. Also it's just more obvious what this value actually is when reading it.

Reply:
Done!